### DISCUSSION TITLE
'Remember Me' function

### DISCUSSION THREAD

**Discussion by notmeta** [ory/kratos#1661] [Q&A] [answered]
Hey there,

I'm currently implementing login in my Next.js app.

I think I know the answer already but thought I'd get confirmation here - is there a 'remember me' function for login?
Looks like it's referenced in 0.4 configuration docs, but no where else.

If it doesn't exist, what's the ETA of such a function?

Thanks :smiley: 

P.S. looking forward to MFA, great work on that so far

**Comment by aeneasr** [comment, marked as answer]
You can use `session.lifetime` to set the lifetime of a session which equals "remember me". Users themselves however can currently not chose if they "want to to stay remembered", see https://github.com/ory/kratos/issues/1416

**Reply by notmeta** [reply]
Thanks for the swift response. Not sure how I didn't find that issue, I tried to search around before asking this!

That issue does raise a good point, is such a function even required? Probably not thinking about it.
> What I've seen is that, after or during an MFA flow, the user may choose "remember this device" which basically sets a cookie after MFA. That allows users to continue to log in without having to re-enter MFA codes again and again.

Is this something that is going to be supported in Ory Kratos?

**Reply by aeneasr** [reply]
It's on the backlog: https://github.com/ory/kratos/issues/1643 :)
